### PR TITLE
Feature/inputs outputs image

### PR DESCRIPTION
**Pull request by sbddesign**
Designed an image to help describe inputs and outputs. Includes  feedback from [this Slack thread](https://bitcoindesign.slack.com/archives/C015856BDME/p1618876627121800).

### CODE REVIEW

**Comment by johnsBeharry** [comment]
Thanks for the work on this Stephen! One thing I would say is that the colored version introduces a 3rd style to the page for illustrations.

I personally like the lined versioned more, but would be interested to know what others think.

Besides that I think the original one was more clear, here are some of my struggles with this one.

Screenshot 2021-04-21 at 00 39 24

File: guide/payments/transactions.md
Comment (johnsBeharry) [inline guide/payments/transactions.md:89]:
```suggestion
Likewise, you need to specify the destination address or addresses for the transaction. These are called *outputs* should there be more bitcoin in the inputs than are needed for the payment, a new address will be created in your wallet for the remaining change, often called a *change output*.
```

Just sneaking in this typo fix.

**Comment by Bosch-0** [comment]
If merged this would close https://github.com/BitcoinDesign/Guide/issues/195 fyi

**Comment by sbddesign** [comment]
> If merged this would close #195 fyi

@Bosch-0 Let's not merge yet; this graphic is confusing people.

**Comment by sbddesign** [comment]
@johnsBeharry I hear you. I have created something that is more complicated than it needs to be. I'm not going to bother describing which Input/Outputs go to which TX, because if the design isn't self-explanatory, then it simply doesn't work. I'll let conversation continue in the Slack thread.

File: guide/payments/transactions.md
Comment (sbddesign) [reply guide/payments/transactions.md:89]: I think the period should stay there. I think it should be:

```
These are called *outputs*. Should there be more...
```

Comment (johnsBeharry) [reply guide/payments/transactions.md:89]: yes, you are correct sir. i blame the lack of sleep.

**Comment by johnsBeharry** [comment]
@sbddesign ye no problem, it's a process. let's roll with the feedback from the slack discussion. then update this PR so we can get it on the site.

**Comment by GBKS** [comment]
Just had to push a minor fix. The mobile SVG included a clip-path, which made the center text not show up on mobile. Something to keep an eye out for in the future.

Issue is that clipping paths are assigned IDs, like "clip0" or "path0". Those are supposed to be unique. Problem is that IDs leak across SVG elements in a document and references get screwed up. So best to check for clip-paths before committing. Also make sure to only use center-aligned strokes. Other stroke types are not supported in SVG and are faked by using clip-paths. Easiest way to test is always to pull it up on a few different browsers/devices and test.

**Comment by GBKS** [comment]
@johnsBeharry as editor-in-chief of this page, could you please do one more review and approve if you're happy?

**Review by pavlenex — APPROVED** [review]
LGMT on desktop.

**Pull request merged**
